LoopKit/Guardrail.swift

```diff
@@ -33,18 +33,16 @@ public struct Guardrail<Value: Comparable> {
     }
 
     public func classification(for value: Value) -> SafetyClassification {
-        precondition(absoluteBounds.contains(value), "Only values within the domain can be classified")
-
         switch value {
-        case absoluteBounds.lowerBound where absoluteBounds.lowerBound != recommendedBounds.lowerBound:
+        case ...absoluteBounds.lowerBound where absoluteBounds.lowerBound != recommendedBounds.lowerBound:
```

Comment:
This change attempts to be less draconian around existing persistent data out there that may violate these new guardrails.  Instead of crashing on the precondition above, it ranks those outside of `absoluteBounds` as `.outsideRecommendedRange(.minimum)` or `.outsideRecommendedRange(. maximum)`